scripts/postgres/schema-v1-postgresql.sql

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+--
+--
+--  http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+--
+
+-- Note: Database and schema creation is not included in this script. Please create the database and
+-- schema before running this script. for example in psql:
+-- CREATE DATABASE polaris_db;
+-- \c polaris_db
+-- CREATE SCHEMA polaris_schema;
+-- set search_path to polaris_schema;
+
+CREATE TABLE IF NOT EXISTS entities (
+    catalog_id BIGINT NOT NULL,
+    id BIGINT NOT NULL,
+    parent_id BIGINT NOT NULL,
+    name VARCHAR(126) NOT NULL,
```

Comment:
Why is name restricted to 126 chars?

Reply:
I think i just got this number from existing Eclipselink, there is also ongoing discussion to increase this to something bigger should i move it to something bigger > 500 ? 

issue : https://github.com/apache/polaris/issues/1195#issue-2928516236

Reply:
made it to TEXT for now, open to changing it, this is mostly to address the case above ^^^ where task cleaner is generating the TASK entity with manifest names in the name.